handsontable/src/shortcuts/recorder.js

```diff
@@ -0,0 +1,157 @@
+import { createKeysController } from './keyController';
```

Comment:
I thought that this part of code is highly related to native browser events handling. I thought that simulating events may be not the best solution for checking how it works and that there may be needed a lot of mocks. I thought that it may be better to check how SM works in e2e tests. WDYT?